HOSTING.md

```diff
@@ -75,8 +75,7 @@ npm i -g pnpm
 IF you want to have accounts, leaderboards and IP bans, you will have to install and set-up PostgreSQL database
 
 ```sh
-sudo apt -y postgresql
-sudo -u postgres initdb --locale=C.UTF-8 --encoding=UTF8 -D /var/lib/postgres/data --data-checksums
+sudo apt install -y postgresql
```

Comment:
Never a fan of putting `-y` in docs. People should always read the proposed changes to their system if they don't know what they're doing.